Docker/CI.Ubuntu22.dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:22.04
+
+# basic dependencies -
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -y && \
+  apt-get install -y 
+
+RUN apt-get install -y curl \
+  pkg-config \
+  findutils \
+  wget
+
+# build dependencies
+RUN apt-get install -y && \
+    apt-get remove -y llvm-12 clang-format-11 clang-tidy-11 && \
```

Comment:
What does the L14 install?

Reply:
It is redundant - it has been removed. Thanks!